### PR TITLE
#161348760 Users should be able to comment on articles

### PR DESCRIPTION
**Pull request by Sulaiman-Mozes**
#### What does this PR do?
 - A user is able to see comments and also comment on an article

#### Description of Task to be completed?
- Display an article with its comments
- Add a comment to an article

#### Any background context you want to provide?
 - A user is able to see comments when he/she is not logged in but must be logged in to comment on an article

#### ScreenShots
image

#### What are the relevant pivotal tracker stories?
[#161348760](https://www.pivotaltracker.com/story/show/161348760)

### CODE REVIEW

File: src/components/Home/Home.scss [hunk 1]
```diff
@@ -14,5 +14,4 @@
     100% {
       transform: rotate(20deg);
     }
-  }
-  
\ No newline at end of file
+  }
```
Comment (judeinno) [inline src/components/Home/Home.scss]: Add new line

File: src/components/comments/comments.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.comment{
+    display: flex;
+    flex-direction: column;
+}
```
Comment (judeinno) [inline src/components/comments/comments.scss]: Also here add a new line

File: .babelrc [hunk 3]
```diff
@@ -6,4 +6,4 @@
   "plugins": [
     "@babel/plugin-proposal-class-properties"
   ]
-}
+}
```
Comment (judeinno) [inline .babelrc:9]: Add a new line here

**Comment by brucemakallan** [comment]
Please add instructions on how to test it manually.

**Review by brucemakallan — COMMENTED** [review]
I wasn't able to see comments after submitting

**Review by judeinno — DISMISSED** [review]
Thanks for working on my changes

**Review by brucemakallan — DISMISSED** [review]

**Review by RachaelNantale — DISMISSED** [review]

**Review by dnuwa — DISMISSED** [review]

**Review by judeinno — DISMISSED** [review]

**Review by dnuwa — DISMISSED** [review]

File: src/components/Profile/profile.scss [hunk 4]
```diff
@@ -84,7 +84,7 @@
    color: white !important;
  }
  .form-control{
-  background-color: turquoise;
+  // background-color: turquoise;
```
Comment (tibetegya) [inline src/components/Profile/profile.scss]: remove the commented out code

File: src/components/comments/index.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+};
+
+export const CommentForm = ({ onSubmitHandler, onChangeHandler }) => (
+  <div className="col-sm-8 offset-2">
+    <h3> Comments </h3>
+    <br />
+    <form onSubmit={onSubmitHandler}>
+      <div className="form-group">
+        <p>Write your comment :</p>
+        <textarea className="form-control" rows="5" id="comment" name="comment_body" onChange={onChangeHandler} required />
```
Comment (tibetegya) [inline src/components/comments/index.js]: this line is too long

File: src/components/Profile/profile.scss (hunk 4 above)
Comment (Sulaiman-Mozes) [reply src/components/Profile/profile.scss]: Fixed

**Review by judeinno — APPROVED** [review]

**Review by brucemakallan — APPROVED** [review]

**Pull request merged**